### ISSUE TITLE
Communications 🌐 OLS-5

### ISSUE THREAD

**Issue by Ismael-KG** [Ismael-KG/A-History-of-Research-Ethics#42]
### Title

OLS-5

### Elevator Pitch

Apply to OLS-5! https://openreview.net/group?id=openlifesci.org/Open_Life_Science/2022/Cohort_5

### Justification

I would love to learn how to properly maintain _A History of Research Ethics_ and maybe share thoughts about making GitHub more inclusive!

### Tasks
- [x] Sign up to OpenReview
- [x] Remember OpenReview password >.<
- [x] Apply by 15-Jan 2022!

**Comment by Ismael-KG** [comment]
Attempt 1 (one) now ready. Will sleep on it and see what I edit tomorrow.

**Comment by Ismael-KG** [comment]
Following template from [OpenReview](https://openreview.net/group?id=openlifesci.org/Open_Life_Science/2022/Cohort_5).

### Title
A History of Research Ethics

### TL;DR
A [GitHub repository](https://github.com/Ismael-KG/A-History-of-Research-Ethics/) and community to engage with people interested in reading, sharing and contributing to [_A History of Research Ethics_](https://www.tiki-toki.com/timeline/entry/1753034/A-History-of-Research-Ethics/).

### Keywords
History of Science, Philosophy of Science, Research Ethics

### Project
#### Describe your project
[_A History of Research Ethics_](https://www.tiki-toki.com/timeline/entry/1753034/A-History-of-Research-Ethics/) is a free, online resource for researchers, governance professionals, and even college students to learn about science and ethics, and be inspired to develop practical tools for the assurance of adequately conducted research.

A key purpose of _A History of Research Ethics_ is to demonstrate the variety of disciplines and backgrounds that research ethics can draw on. In other words, interdisciplinarity is critical for its success. This means both interdisciplinary contributions _and_ adapting to audiences from diverse fields and sectors.

By embracing the collaborative nature of GitHub and OLS’s open science community, I expect to take _A History of Research Ethics_ to its next stage of development.

### Problem
#### What problem(s) are you solving with this project?
One significant problem is solved by _The Timeline_: the myth that research ethics pertains to only a few heinous medical investigations conducted in the twentieth century. This is solved by providing an interactive timeline on [Tiki-Toki]( https://www.tiki-toki.com/timeline/entry/1753034/A-History-of-Research-Ethics/) that illustrates very diverse events that (i) are relevant to research ethics and (ii) are spread out through history, as far back as the fourth millennium BCE..

It is key to this project that research ethics is acknowledged as a complex field by communities that are equally diverse. I am here speaking of _epistemic diversity_, or diversity of knowledge domains. Generally, the complexity of research ethics might be acknowledged by philosophers; and the depth of the history of science acknowledged by anthropologists and historians. The interactive format of Tiki-Toki enables these ideas to be conveyed to much wider audiences.

### Achievements
#### What concrete achievements do you want to accomplish during the OLS program?
Key achievements include:
* Gathering feedback, both on the content of _A History of Research Ethics_ and on the way GitHub is used as a repository. _(Nb.: this is not a “SMART” goal.)_
* Create a sustainability plan to ensure that future contributors have clear “protocols” to follow, but also that contributors needn’t have any knowledge of GitHub – it must all be spelled out clearly and without jargon. The protocols currently in place are [here]( https://github.com/Ismael-KG/A-History-of-Research-Ethics/blob/main/Protocols.md).
* Establish clear links between story categories and potential audiences. Currently captured in “[Protocol 7: Public Engagement](https://github.com/Ismael-KG/A-History-of-Research-Ethics/blob/main/Protocols.md#protocol-7-public-engagement), Intended Audiences”, I will work with OLS to understand how different audiences can gain from _The Timeline_ and create more suitable protocols.
* Gain maybe two contributors, as defined by [Protocol 3](https://github.com/Ismael-KG/A-History-of-Research-Ethics/blob/main/Protocols.md#Protocol-3-Contribution-Types).

### Open
#### What inspires you to work openly in research?
Very roughly, _epistemology_ is the field that studies what knowledge is, how it is shaped, its purpose and its evolution. For decades, this philosophical field has challenged the notion that research is conducted by “lone geniuses”. Rather, the field of _social epistemology_ has gained traction. Social epistemology argues that knowledge is produced by and for social collectives.

Standing on the shoulders of great thinkers such as Ian Hacking, Miriam Solomon and Robert Merton, I draw on the open science movement to conclude that collaborative methods of knowledge-production can also be conducted openly. This is so that those who collaborate can learn together about the content they contribute to, and methods for collaboration. The goal is for collaborators to become advocates for social forms of epistemology.

### Challenges
#### What challenges have you faced in working in open science?
Two main challenges, although intricately linked. Firstly, the technical barriers. I have only worked in one open science community, and it was designed around its tool for collaboration: GitHub. Whilst I have come to terms with some of GitHub’s functions in recent months (and I promise I can use it for OLS-5!), I
struggled in many ways to engage with the creation of the community's metascience book.

Secondly, I have faced an important cultural barrier. By relying on traditional open science tools such as GitHub, it can be difficult to be understood by fellow collaborators who are perfectly comfortable with the tool. I found that they struggled to empathise with the fact that I would spend so much effort in making
small changes that I had no capacity to make more significant contributions. In other words, I have struggled with bringing diverse fields into otherwise very code-intensive communities.

### Learning
#### What are your learning goals in the Open Life Science program?
I want to learn how to engage with diverse and technical communities. I will take advantage of the OLS format to feel a part of a welcoming community, network with people with practical know-how in open science, and embrace GitHub once and for all!

Importantly, I want to improve in my social skills. I am generally quite the introvert. On top of that, I have been very isolated in recent months, so “re-learning” social skills will be of great value, and I think the OLS-5 cohort will be a kind group for me to engage with.

### Mentor
#### What do you expect from a mentor?
Kindness, patience, empathy, curiosity and structure. I would like a mentor who is interested in metascience, willing to advocate for my work and prepared to help me network. (Networking is a key skill I have lost during the pandemic!)

I also don’t have a PhD, which I have found to be detrimental to my credibility in the past. I would welcome a mentor from industry, or with know-how both within and without academia, but this is only a nice-to-have. I will be grateful to whomever becomes my mentor!

**Comment by Ismael-KG** [comment]
And now wait and see!
